### DISCUSSION TITLE
How does OTT save the username for guest users?

### DISCUSSION THREAD

**Discussion by Issykul** [dyc3/opentogethertube#681] [Q&A] [answered]
Hey Community, i am not saving any cookies. I set my Browser to always delete everything when closing my Browser, but OTT still is able to save my custom guest username. How does it save it?

**Comment by dyc3** [comment, marked as answer]
OTT uses [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) to save your auth token. The username is associated with your auth token. This is also how client side settings are stored. If you want to clear your token, open your js console and run `localStorage.removeItem('token')`.
